README.md

```diff
@@ -1,2 +1,126 @@
+- A source of inspiration and a driver of innovation.
+- A designated place for writing down bright ideas and suggestions.
+- A place to share, discuss and review trending tech topics and new promising
+tools.
+
+**Everyone is welcome to participate:**
+
+- [Start](https://github.com/hyperoslo/hyper-radar/issues/new)
+a discussion if you have something on your mind.
+- Choose an appropriate category and write a review if you want to put some
+light on any tech related topic.
+
+## 📖 Table of Contents
+
+The project's structure will gradually change over time and is always open for
+adjustments. The main goal is to make it easy and intuitive to navigate through
+and find a relevant information.
+
+* [Lab](#areas)
+* [Stage](#tools)
+* [Inspiration](#inspiration)
+* [Contributing](#contributing)
+* [Author](#author)
+* [License](#license)
+
+## 🔭 Lab
+
+We all have our own areas of expertise and preferences, something we're best
+at and can talk with excitement and enthusiasm for hours. And there is always
+something we're missing because we're so used to seeing what we already know
+and are comfortable with. The technology is not staying still, and from time to
+time it's healthy to force ourself into those uncomfortable spheres where we
+can start challenging our thinking.
+
+We need to know where our focus should be. Unexpected customer requests are
+inevitable and can't be foreseen, but what we can do is to be well prepared and
+know the right answers. It's hard to know everything without becoming
+a "Master of None", but it helps to have an overview of what is possible and
+what is not, without digging too deep into implementation details.
+
+We live to break down and solve complex tasks by creating practical,
+maintainable, and scalable solutions. And this is the reason we love great
+development tools. But the right tools should be built around the actual needs,
+not the other way around. There is no "silver bullet", so remember that the
+main goal is to find out the most elegant and efficient way for solving a given
+problem, at certain circumstances.
+
+Share your experience to help in our community to stay sober and make the
+right decision when it comes to choosing a new library or programming
+language everyone around is talking about. Try to ask yourself the following
+questions when you write a review:
+
+- What all the fuzz is about?
+- What is it for?
+- How mature the tool is?
+- How could it be applied to real-world problems?
+- What are the advantages and disadvantages comparing to existing solutions?
+- How relevant is it for us and our customers?
+
+## 🎤 Stage
+
+Have you recently been a speaker at a local meetup or at a conference?
+Open the [Stage](https://github.com/hyperoslo/hyper-radar/blob/master/Stage)
+folder and share your keynote, so everyone can enjoy the slides and re-visit it
+on any day, at any time.
+
+## 🦄 Inspiration
+
+Use (https://github.com/hyperoslo/hyper-radar/blob/master/inspiration.md) to
```

Comment:
I don't know yet how useful it is to have a file with a list of links, but we'll see how it goes, maybe  we don't need it at all, or maybe it makes sense to slip it up to multiple files.